components/prism-turtle.js

```diff
@@ -0,0 +1,31 @@
+Prism.languages.turtle = {
+	'comment': {
+		pattern: /#.*/,
+		greedy: true
+	},
+	'multiline-string': {
+		pattern: /"""(?:[^"]"?"?)*"""|'''(?:[^']'?'?)*'''/,
+		greedy: true,
+		alias: 'string',
+		inside: {
+			'comment' : {
+				pattern: /#.*/
+			}
+		}
+	},
+	'string': {
+		pattern: /"(?:[^\\"\r\n]|\\.)*"|'(?:[^\\'\r\n]|\\.)*'/,
+		greedy: true
+	},
+	'url': {
+		pattern: /<[^\s<>]*>/,
```

Comment:
Just an idea but wouldn't it look good if the `<>` inside the URL were highlighted as `punctuation`?

Reply:
Yes, looks good. Also added `@` as punctuation in language tag.